rwengine/src/script/ScriptMachine.cpp

```diff
@@ -81,7 +81,7 @@ void ScriptMachine::executeThread(SCMThread &t, int msPassed)
 				hasExtraParameters = false;
 				break;
 			case TInt8:
-                parameters.back().integer = _file->read<std::uint8_t>(pc);
+                parameters.back().integer = _file->read<std::int8_t>(pc);
```

Comment:
Good catch. I've been trying to sneak clang-format changes here and there, this is a good candidate.